packages/vega-typings/types/runtime/runtime.d.ts

```diff
@@ -0,0 +1,370 @@
+import { Config, Format } from '../spec';
+import { EventType, WindowEventType, Transforms, Binding as SpecBinding } from '..';
+// All references to source code are from the vega-parser package
+
+export interface Runtime {
+  description: string;
+  operators: Operator[];
+  // Event streams that Vega is listening to. Often DOM events.
```

Comment:
Let's use /**...*/ comments so that we get previews in vscode.

Reply:
Sounds good, will update